src/components/include/policy/policy_external/policy/policy_manager.h

```diff
@@ -532,14 +532,6 @@ class PolicyManager : public usage_statistics::StatisticsManager {
    */
   virtual const PolicySettings& get_settings() const = 0;
 
-  /**
```

Comment:
@ZhdanovP 
1) separate removing unused funcs to separate PR.
2) please test building and unittests with 2 combinations of EXTENDED_POLICY
 - PROPRIETARY - >> default value. already tested be CI(build server) - success
 - EXTERNAL_PROPRIETARY

Reply:
I've tried to tests EXTERNAL_PROPRIETARY and REMOTE_CONTROL set to OFF
it fails.
But obviously your PR does not influe on it